addons/im_livechat/static/tests/embed/autopopup_tests.js

```diff
@@ -19,14 +19,16 @@ QUnit.test("persisted session", async () => {
     const channelId = pyEnv["discuss.channel"].create({
         channel_member_ids: [
             Command.create({ partner_id: pyEnv.adminPartnerId }),
-            Command.create({ guest_id: guestId }),
+            Command.create({ guest_id: guestId, is_minimized: true }),
         ],
         channel_type: "livechat",
         livechat_channel_id: livechatChannelId,
         livechat_operator_id: pyEnv.adminPartnerId,
     });
-    const [channelInfo] = pyEnv.mockServer._mockDiscussChannelChannelInfo([channelId]);
-    cookie.set("im_livechat_session", JSON.stringify(channelInfo));
+    cookie.set(
+        "im_livechat.saved_state",
+        JSON.stringify({ threadData: { id: channelId, model: "discuss.channel" }, persisted: true })
```

Comment:
It will always be a `discuss.channel`. This information doesn't need to be stored.

Reply:
That does not take up much space and it allows us to insert the data as is: `store.Thread.insert(data)`. Without it, we should add it each time we need to use `get/insert` which would be cumbersome.